docs/changes/units/11691.feature.rst

```diff
@@ -0,0 +1,2 @@
+Each physical type is added to the physical module in ``units`` along with a
```

Comment:
```suggestion
Each physical type is added to ``astropy.units.physical`` along with a
```
Slight edit.  Another possibility would be to add back ticks around "physical" in "physical module".